packages/analytics/tsconfig.json

```diff
@@ -1,15 +1,11 @@
 {
+  "extends": "../../configs/typescript/tsconfig.package-build.json",
```

Comment:
What's the difference between `tsconfig.build.json` and `tsconfig.package-build.json`? Maybe we can describe these configurations in the README or another place.

Reply:
The purpose of those files are to be used only when running `yarn types` command, which will be used by the pipeline when we migrate the codebase fully to typescript. This is because the default tsconfig.json enables typings of test files and we do not want to have them outputted when we generate a new version of the package.

I was thinking of naming them exactly as `tsconfig.package-build.json` but the reason I didn't pick that name is that it might get more difficult to navigate in your IDE when you have many files with the same name opened.

What do you think? Do you have a better naming strategy?

Reply:
Naming is always a problem 🤣  to be honest, I don't have a better suggestion I was trying to understand the purpose of this file.